crates/cxx-qt-gen/src/generator/rust/signals.rs

```diff
@@ -203,6 +265,19 @@ mod tests {
                 }
             },
         );
+        assert_tokens_eq(
+            &generated.cxx_mod_contents[7],
+            quote! {
+                unsafe extern "C++" {
+                    #[doc = "Connect the given function pointer to the signal "]
+                    #[doc = "baseName"]
+                    #[doc = ", so that when the signal is emitted the function pointer is executed."]
+                    #[must_use]
+                    #[rust_name = "on_base_name"]
```

Comment:
the name ends up being odd here as it's the `cxx_name`.

Reply:
But this also happened before the signal name internally anyway, so probably a separate change.

Reply:
Fixed :-)